pom.xml

```diff
@@ -47,17 +36,21 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.1</version>
                 <configuration>
-                    <argLine>
```

Comment:
why are you remove AspectJ configuration?

Reply:
why is it needed? it worked without it....

Reply:
to add `@Step` and `@Attachment` annotations support. Otherwise you will have to steps and attachments in the report.

Reply:
o_O :) oookay, it wasn't clear to me.